frontend/src/components/mine/Incidents/MineIncidentTable.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Table } from "antd";
+import CustomPropTypes from "@/customPropTypes";
+import NullScreen from "@/components/common/NullScreen";
+import { formatDate } from "@/utils/helpers";
+
+const propTypes = {
+  incidents: PropTypes.arrayOf(CustomPropTypes.incident).isRequired,
+  followupActions: PropTypes.arrayOf(CustomPropTypes.incidentFollowupType).isRequired,
+};
+
+const columns = [
+  {
+    title: "Incident Report No.",
+    dataIndex: "mine_incident_report_no",
+  },
+  {
+    title: "Incident Time",
+    dataIndex: "incident_timestamp",
+  },
+  {
+    title: "Reported By",
+    dataIndex: "reported_by",
+  },
+  {
+    title: "EMPR Action",
+    dataIndex: "followup_action",
+    render: (action) => (
+      <div title="followup_action">{action ? action.description : "error..."}</div>
```

Comment:
I think we have error string constants. @mmumby ?

Reply:
yeah there are constants, with this being a table I'm thinking the null view should be "N/A" or "-" which is consistent with the other tables. We don't use "error..." anywhere else.

Reply:
This should never happen, but we didn't want the app to crash if it did.... so is there more discussion we want to have to handle and show subtle errors? or should we just keep it simple and make it '-'?

Reply:
We already have the three letter code, i'm just going to display that if something goes wrong.